src/cdk/drag-drop/drag-ref.ts

```diff
@@ -884,8 +884,9 @@ export class DragRef<T = any> {
     // otherwise iOS will still add it, even though all the drag interactions on the handle
     // are disabled.
     if (this._handles.length) {
-      this._rootElementTapHighlight = rootElement.style.webkitTapHighlightColor || '';
-      rootElement.style.webkitTapHighlightColor = 'transparent';
+      const rootStyles = rootElement.style as any;
```

Comment:
I'm generally not a fan of `any`, but it's acceptable in this case since the type for `CSSStyleDeclaration` is wrong. Here are a two suggestions that are more type safe than `any`.

 - `CSSStyleDeclaration & { [key: string]: string|null|undefined }`
 - `CSSStyleDeclaration & { webkitTapHighlightColor?: string|null}`